### PR TITLE
Website clean-up

### PR DESCRIPTION
**Pull request by silverailscolo**
Links to old groups, hosting updated or converted to plain text
HTML clean-up, spelling, unused images

### CODE REVIEW

File: install/MacOSXcomm.html [hunk 1]
```diff
@@ -49,7 +49,7 @@
 <!-- /Header -->
 
   <div class="nomenu" id="mBody">
-    <div id="mainContent">
```
Comment (bobjacobsen) [inline install/MacOSXcomm.html]: I think the “div mainContent” here is part of the CSS formatting of the page. (I’ve seen lots of pages missing the corresponding close div)

File: manual/3-0_DecoderPro/Installing_JMRI.shtml [hunk 2]
```diff
@@ -63,24 +62,21 @@
...
-<a href="http://sourceforge.net">
-<IMG src="images/sflogo.png" width="120" height="30" border="0" alt="SourceForge Logo" hspace="10" vspace="5" align="top"></a>
+  <a href="http://www.tagadab.com">
+    <img src="https://www.tagadab.com/sites/default/files/logo-tagadab-nostrap.png" height="28" width="103"
+         border="0" alt="Tagadab Logo"> </a>
 <br>
 </div>
 
-<!--#include virtual="/Footer.shtml" -->
-</div>
```
Comment (bobjacobsen) [inline manual/3-0_DecoderPro/Installing_JMRI.shtml]: Is this one perhaps missing a “div mBody”? We don’t have this structure well documented, but there are examples in /templates

Comment (silverailscolo) [reply manual/3-0_DecoderPro/Installing_JMRI.shtml]: Especially these old _Manual_ pages don't follow the full CSS structure very well. Will try to clean them up a bit (anybody reading them?)

**Comment by bobjacobsen** [comment]
>  Will try to clean them up a bit (anybody reading them?)

Probably not many. And so probably not worth a lot of time.

As long as the pages I commented on display correctly, they probably don’t need any additional time.

File: install/MacOSXcomm.html (hunk 1 above)
Comment (silverailscolo) [reply install/MacOSXcomm.html]: Done

File: manual/3-0_DecoderPro/Installing_JMRI.shtml (hunk 2 above)
Comment (silverailscolo) [reply manual/3-0_DecoderPro/Installing_JMRI.shtml]: Done

**Pull request merged**